README.md

```diff
@@ -51,16 +51,36 @@ about this tool as well as e2e testing generally.
 
 Create a PR in this repo to add/update/remove a job or suite. Specifically
 you'll need to do the following:
-* Create an entry in [`jobs/config.json`] for the job
-  - If this is a kubetest job create the corresponding `jobs/env/FOO.env` file
-  - It will pick a free project from [boskos](/boskos) pool by default, or
-  - You can also set --gcp-project=foo in [`jobs/config.json`] for a dedicated project, make sure the project has the right [IAM grants](jenkins/check_projects.py)
+* Add the job to the appropriate section in [`config/jobs`](config/jobs)
+  - Directory Structure:
+    - In general we follow config/jobs/org/repo/name-of-config-file.yaml
+    - For Kubernetes Repo we also allows config/jobs/kubernetes/sig-foo/name-of-config-file.yaml
+    - We use basename of the config name as a key in the prow configmap, so the name of your config file need to be unique across the config subdir.
```

Comment:
We need a presubmit to enforce this

Reply:
we enforce this in config agent.

Reply:
(this is a general comment and not a blocker for this PR)

Reply:
I mean, our presubmit loads the entire config, and I added a check at config loading time: https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L356

Reply:
OK, if you find yourself explaining the naming convention to everyone then maybe we can add the other presubmit too.

Reply:
I deliberately allowed https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml

so whoever add a config.yaml again, presubmit will rant

Reply:
the error might suggest how to ensure the name is unique though (a bash line to get the basenames perhaps?) /shrug

Reply:
I can print out the full path of the conflicting files :-)